test/Api.IntegrationTest/SecretsManager/SecretsManagerOrganizationHelper.cs

```diff
@@ -25,13 +25,22 @@ public SecretsManagerOrganizationHelper(ApiApplicationFactory factory, string ow
         _ownerEmail = ownerEmail;
     }
 
-    public async Task<(Organization organization, OrganizationUser owner)> Initialize(bool useSecrets, bool ownerAccessSecrets)
+    public async Task<(Organization organization, OrganizationUser owner)> Initialize(bool useSecrets, bool ownerAccessSecrets, bool organizationEnabled = true)
```

Comment:
Minor question: would it be safer to default this to false?

Reply:
Spoke with Thomas Avery and we decided that it's probably best to explicitly pass in the value instead of defaulting to either false/true, thank you for pointing this out 😄